QDMA/DPDK/drivers/net/qdma/qdma_rxtx.c

```diff
@@ -977,6 +992,12 @@ uint16_t qdma_recv_pkts_st(struct qdma_rx_queue *rxq,
 
 	if (nb_pkts_avail == 0) {
 		PMD_DRV_LOG(DEBUG, "nb_pkts_avail = 0\n");
+
+#ifdef RTE_LIBRTE_SPIRENT
+        /* Replenish descriptors even when there is nothing to receive (overflow condition) */
+        qdma_sp_rearm_pkts_st(rxq);
+#endif
+
```

Comment:
This is the fix frr CIPMAV-251.
Need to check and refill the descriptor queue with buffers even if we have not received any frames otherwise a buffer exhaustion will leave this in a state where no frames can ever be received again.